packages/react-query/src/internals/context.tsx

```diff
@@ -166,6 +168,26 @@ export interface TRPCContextState<
     options?: InvalidateOptions,
   ): Promise<void>;
 
+  /**
+   * @link https://react-query.tanstack.com/reference/QueryClient#queryclientresetqueries
+   */
+  resetQueries<
+    TPath extends keyof TRouter['_def']['queries'] & string,
+    TInput extends inferProcedureInput<TRouter['_def']['queries'][TPath]>,
+  >(
+    pathAndInput?: [TPath, TInput?] | TPath,
+    filters?: ResetQueryFilters,
+    options?: ResetOptions,
+  ): Promise<void>;
+
+  /**
+   * @link https://react-query.tanstack.com/reference/QueryClient#queryclientresetqueries
+   */
+  resetQueries(
+    filters?: ResetQueryFilters,
+    options?: ResetOptions,
+  ): Promise<void>;
+
```

Comment:
I don't think we should support this. What do you think @KATT?

Reply:
Just had a place where we really needed this functionality, if there are any alternatives maybe we could use them, but could not found any :)

The functionality we was looking for was the ability to reset a query without an observer.

Reply:
How come you don't think we should support it, @sachinraja?

Reply:
I meant specifically for the v9, interop API. We should definitely add the ability to reset queries. Are we still going to add new features for the interop API?

Reply:
Nono, not for v9!